nncf/scopes.py

```diff
@@ -65,9 +65,9 @@ def __init__(self,
             names of ignored nodes.
         :param types: List of ignored operation types.
         """
-        self.names = names
-        self.patterns = patterns
-        self.types = types
+        self.names = names if names is not None else []
+        self.patterns = patterns if patterns is not None else []
+        self.types = types if types is not None else []
```

Comment:
Pay attention to `calibreate.py` and `test_quantize_conformance.py` as they could expect `None` here

Reply:
I've double checked and didn't find anything that is related to None value.

Reply:
@alexsu52, is it ok to have `[]` instead of `None` as default values in `IgnoredScope`?

Reply:
What benefit from these changes?

Reply:
If you want to add ignore_scope there is no need to check on None